packages/code-connect/components/Modal/BasicModal.figma.tsx

```diff
@@ -0,0 +1,21 @@
+import figma from '@figma/code-connect';
+import { Modal } from '@patternfly/react-core';
+
+figma.connect(
+  Modal,
+  'https://www.figma.com/design/aEBBvq0J3EPXxHvv6WgDx9/PatternFly-6--Components-Test?node-id=2937-158&t=IzSunfrnw18ti37Y-11',
+  {
+    props: {
+      size: figma.enum('Size', {
+        Small: 'small',
+        Medium: 'medium',
+        Large: 'large'
+      }),
+      children: figma.children('*')
+    },
+    example: (props) => (
+      // Documentation for Modal can be found at https://www.patternfly.org/components/modal
+      <Modal variant={props.size}>{props.children}</Modal>
```

Comment:
This will have the same issues as the [AboutModal](https://github.com/patternfly/patternfly-react/pull/11827/files#r2140919712) where the Modal will not appear unless it has isOpen={true} but really they probably want a button which will allow them to open/close the modal...